Gemfile

```diff
@@ -2,24 +2,24 @@
 # If you modify this file manually all comments and formatting will be lost.
 
 source "https://rubygems.org"
-gem "logstash-core", "5.0.0.alpha1.snapshot1"
-gem "logstash-core-event-java", "5.0.0.alpha1.snapshot1"
-gem "logstash-core-plugin-api", "1.4.0"
+gem "logstash-core", "5.0.0.alpha1.snapshot2"
+gem "logstash-core-event-java", "5.0.0.alpha1.snapshot2"
+gem "logstash-core-plugin-api", "1.5.0"
 gem "file-dependencies", "0.1.6"
 gem "ci_reporter_rspec", "1.0.0", :group => :development
-gem "simplecov", :group => :development
-gem "coveralls", :group => :development
+gem "simplecov", ">= 0"
```

Comment:
ergh, I ran rake test before I've made the change to the gemfile will revert that.